corehq/form_processor/utils/sql.py

```diff
@@ -61,6 +61,10 @@ def form_adapter(form):
         form.deleted_on,
         form.deletion_id,
         form.server_modified_on,
+        form.time_start,
+        form.time_end,
+        form.commcare_version,
+        form.app_version
```

Comment:
did you check this order? it should be the same order as appears in the SQL table definition:
```
\d form_processor_xforminstancesql
```

Reply:
Thanks, I have updated, and added a note.